config-editor/README.md

```diff
@@ -0,0 +1,7 @@
+# Annotator for Clinical Data Configuration Editor Bundle
+
+This bundle contains the scripts and charts for deploying the configuration editor to your environment.
```

Comment:
Suggestion here is uppercase "Configuration Editor" for consistency with the docs/proper naming for the offering.